account_banking_payment_export/wizard/payment_order_create.py

```diff
@@ -45,9 +46,33 @@ def default_get(self, field_list):
     def extend_payment_order_domain(self, payment_order, domain):
         if payment_order.payment_order_type == 'payment':
             domain += [('account_id.type', 'in', ('payable', 'receivable')),
-                       ('amount_to_pay', '>', 0)]
+                       ('credit', '>', 0)]
```

Comment:
@pedrobaeza can you point me where? On my local copy of the 8.0 branch I can't find amount_to_pay anywhere in the code. I may have missed something.